src/duration-picker/DurationCard.js

```diff
@@ -101,9 +101,9 @@ export default class DurationCard extends Component {
         if (!currentUnitBlock) return null;
 
         return (
-            <div className="duration-picker">
+            <div className="param-row">
```

Comment:
Nope, this is a step backwards.
duration-picker, something-else-picker {
...
}
is much better than param-row {
}
Since then you can also style separately.